modules/installation-requirements-user-infra.adoc

```diff
@@ -69,8 +69,9 @@ ifdef::ibm-z[]
 
 To install on IBM Z under z/VM, you require a single z/VM virtual NIC in layer 2 mode. You also need:
 
-*   A direct-attached OSA or RoCE network adapter
-*   A z/VM VSwitch set up. For a preferred setup, use OSA link aggregation.
+* A direct-attached OSA network adapter
```

Comment:
Add periods to the end of these list items.

Reply:
Fixed in e4bec360f9254dfcdbc363a6fa918c278ecab916